Enigmatry.Entry.Blueprint.Infrastructure/Data/PopulateCreatedUpdatedInterceptor.cs

```diff
@@ -36,21 +36,15 @@ public override ValueTask<InterceptionResult<int>> SavingChangesAsync(DbContextE
 
     private void PopulateCreatedUpdated(DbContext dbContext)
     {
-        var currentUser = _currentUserProvider().User;
-        if (currentUser == null)
-        {
-            // TODO: Discuss throwing an exception here; Currently seeding data in the integration tests would fail because of it
```

Comment:
This was a problematic solution and it was now fixed with SystemUserProvider and TestUserProvider.